<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientRegistrationPage</class>
 <widget class="QWidget" name="PatientRegistrationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Patient Information</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_ptntName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Person</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_ptntSurname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Doe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_ptntId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>8801012345678</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_DOB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date of Birth</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_gender">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>F</string>
    </property>
    <property name="maxVisibleItems">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_ptntCell">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>081 2345 678</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cellphone</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Address</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Complex Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Postal Code</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="et_addLn1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>5 Complex Block</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="et_city">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Pretoria</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="et_addLn2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>2 First Street</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="et_postal">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0001</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_ptntEmail">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>person.doe@gmail.com</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Appointment Details</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Facility</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vaccination Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_facility">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Steve Biko Hospital</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dt_vaxdate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ptntRegSubmit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>B o o k</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_missingnfo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All the Fields are compulsory, please fill in all of them.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ptntRegErr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Something went wrong please contact the system administrator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
